packages/instant/src/components/ui/container.tsx

```diff
@@ -29,11 +28,7 @@ export interface ContainerProps {
     zIndex?: number;
 }
 
-const PlainContainer: React.StatelessComponent<ContainerProps> = ({ children, className }) => (
-    <div className={className}>{children}</div>
-);
-
-export const Container = styled(PlainContainer)`
+export const Container = styled<ContainerProps, 'div'>('div')`
```

Comment:
I prefer the syntax:`export const Container = styled.div <ContainerProps>` , although I'm curious if they are equivalent?

Reply:
Does that one work? It seemed to be producing an `any` type?

Reply:
Looks like it's working on my side: 

![image](https://user-images.githubusercontent.com/210914/47445220-455e1180-d76d-11e8-9b3d-b13405aa6b68.png)

Reply:
@steveklebanoff are the interpolated `props` also typed?

Reply:
Yea when I use this I find that it cannot infer the type of `props`